setup.cfg

```diff
@@ -17,10 +10,22 @@ oteapi.parse =
   #oteapi_dlite.application/n-triples = oteapi_dlite.strategies.parse:DLiteParseStrategy
   #oteapi_dlite.text/turtle = oteapi_dlite.strategies.parse:DLiteParseStrategy
   #oteapi_dlite.text/csv = oteapi_dlite.strategies.parse:DLiteParseStrategy
+  #oteapi_dlite.image/gif = oteapi_dlite.strategies.parse_image:DLiteImageParseStrategy
```

Comment:
Remove commented-out code

Reply:
I would rather like to uncomment them and instead removing the vendor-specific metiatypes below.